obspy/io/alsep/core.py

```diff
@@ -16,16 +17,29 @@
 from .wt.tape import WtnTape, WthTape
 
 
-def _is_pse(filename):
+def _read_header(file):
+    """Reads the header of the given ALSEP file"""
+    with open_bytes_stream(file) as stream:
+        # np.fromfile accepts file path on-disk file-like
+        # objects. In all other cases, use np.frombuffer:
+        try:
+            return np.fromfile(file, dtype='u1', shape=16)
+        except Exception:
```

Comment:
this was actually a bug, thanks @megies : `shape` is a typo and should be `count`. Tests were passing because of the bare Except.
Fixed now with `count` instead of `shape` and `UnsupportedOperation` instead of bare Except